op-e2e/system/fees/fees_test.go

```diff
@@ -138,7 +138,7 @@ func testFees(t *testing.T, cfg e2esys.SystemConfig) {
 
 	baseFeeRecipient := predeploys.BaseFeeVaultAddr
 	if sys.RollupConfig.IsCel2(sys.L2GenesisCfg.Timestamp) {
-		baseFeeRecipient = addresses.GetAddresses(cfg.L2ChainIDBig()).FeeHandler
+		baseFeeRecipient = addresses.FeeHandlerAddress
```

Comment:
Why is this coming in? This looks like we're stepping back.

Reply:
So this is coming from the revert of a previous op-geth update.

That update brought in a refactor of the addresses, which required changes in the optimism code. The release branch for op-geth does not contain that refactor.

So options are:
* Update the op-geth release branch by cherry picking the address refactor onto the release branch and then create a new tag for that and then update this pr with the newly tagged op-geth.
* Or just use the current release of op-geth and change the code in the optimism release to use the old addresses.

I chose the second approach since it seemed like the quickest route to having a release of op-geth and a corresponding release of optimism. But let me know what you think.

Reply:
If we didn't have the release branches, we could have just tagged an op-geth commit from celo10 and then updated optimism with that release and tagged it. This seems a lot more straightforward to me.

The downside is that people would have to look for the latest tag to update to, rather than just pulling a branch to get the latest release.